deploy/helm/charts/templates/zfs-controller.yaml

```
@@ -118,7 +118,7 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: OPENEBS_IO_INSTALLER_TYPE
-              value: "zfs-localpv-helm"
+              value: "{{ .Values.analytics.installerType }}"
```

ensure it's not a breaking change when using old values yaml

```suggestion
              value: "{{ default .Values.analytics.installerType "zfs-localpv-helm" }}"
```

---

Doesn't seem to work.

---

```{{ if (not (hasKey .Values.analytics "installerType")) }}zfs-localpv-helm{{ else }}{{ .Values.analytics.installerType }}{{ end }}```

Had to do this.